packages/test/.mocharc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Using `ts-node` works? Does this then necessitate changes in `package.json`, i.e. a dependency + changing how running the tests is invoked?

Reply:
Removed all the ts-node stuff, testing still works.

Reply:
After changing package.json and doing install I get `Couldn't find any versions for "@lionweb/core" that matches "0.5.0-beta.9"`

Reply:
> After changing package.json and doing install I get `Couldn't find any versions for "@lionweb/core" that matches "0.5.0-beta.9"`

Solved now, probably something old hanging around.